augur/augur_client.py

```diff
@@ -181,11 +181,9 @@ def filter_blocks(self, start_block, end_block, filter, handler,
     '''
     self._require_is_open()
     if (start_block < 0) or (end_block < 0) or (increment < 0):
-      raise IOError('Block numbers and increments should be greater than 0.')
+      raise IOError('Block numbers or increment cannot be less than 0.')
```

Comment:
The error messages still don't match the check being performed. Ensure that they are consistent.

Reply:
Is the issue here that the error message does not identify which parameter is raising the exception? Instead of having one check for all 3 parameters I changed it to each parameter raising their own exception.

Reply:
It's a matter of being very clear and precise in your messages. You say ```start_block``` must be less than ```end_block```, but you only check if ```start_block``` is greater than ```end_block```. Pick a set of semantics and ensure it's applied consistently throughout the entire function.